cvm/quantization/dataset.py

```diff
@@ -6,15 +6,25 @@
 from gluoncv.data.transforms.presets.yolo import YOLO3DefaultValTransform
 from gluoncv.utils.metrics.voc_detection import VOC07MApMetric
 import numpy as np
+import requests
+import shutil
+import tarfile
 
 import os
 import math
 import pickle
 
 # max value: 2.64
-def load_voc(batch_size, input_size=416):
+def load_voc(dataset_dir, batch_size, input_size=416):
```

Comment:
Suggest not to change the interface of loading dataset, which may be used by another program except main.py. And it's the responsibility of dataset.py to determine the location of a dataset instead of the main.py.
```suggestion
dataset_dir = "~/.cvm"
def load_voc(batch_size, input_size=416):
    # used dataset_dir as before.
```